ludwig/features/numerical_feature.py

```diff
@@ -267,7 +270,21 @@ def postprocess_predictions(
             skip_save_unprocessed_output = True
 
         if PREDICTIONS in predictions and len(predictions[PREDICTIONS]) > 0:
-            postprocessed[PREDICTIONS] = predictions[PREDICTIONS].numpy()
+            if metadata['preprocessing']['normalization'] is not None:
+                normalization_type = metadata['preprocessing']['normalization']
+                NumericTransformer = get_from_registry(
+                    normalization_type,
+                    numeric_transformation_registry
+                )
+
+                numeric_transformer = NumericTransformer(**metadata)
+                values_to_return = numeric_transformer.inverse_transform(
+                    predictions[PREDICTIONS].numpy()
+                )
+            else:
+                values_to_return = predictions[PREDICTIONS].numpy()
+
+            postprocessed[PREDICTIONS] = values_to_return
```

Comment:
We may further avoid this if (and the corresponding one in preprocessing) by having an `IdentityTransformer` in the registry associated with a `None` key, and obtaining the transformer by:
```
NumericTransformer = get_from_registry(
                      metadata['preprocessing'].get('normalization', None),
                      numeric_transformation_registry
                  )
```

Reply:
done